JavaApplication6/src/javaapplication6/conditions.java

```diff
@@ -0,0 +1,116 @@
+                + "of the word perfect enter the number 100. To exit type 1.");
+        code=keyboard.nextInt();
+        
+    switch (code)
+    {    
+        case 10:
+            System.out.println("Enter your street number");
+            streetNumber = keyboard.nextInt();
+            
+            if (streetNumber < 100)
+            {
+                System.out.println("Simon says to ten jumping jacks.");
+            }
+            else if (streetNumber > 600)
+            {
+                System.out.println("Go and get yourself a candy bar.");
+            }
+            break;
+        
+        case 15:
+            System.out.println("Enter your favorite food that you must eat.");
+            numberOfMeals=keyboard.nextInt();
+            
+        
+            if (numberOfMeals >= 3)
+            {
+                System.out.println("Looks like you are on the right track!");
+            }
+            else if (code < 0) 
+            {        
+                System.out.println("You need to start eating more pizza.");
+            } 
+            break;
+        case 20: 
+            System.out.println("Please enter your age, be honest.");
+            yourAge=keyboard.nextInt();
+            
+        
+            if (yourAge >= 19)
+            {
+                System.out.println("Today is your lucky day!");
+        }
+            else if(yourAge <= 18)
+            {  
+                System.out.println("You will not be lucky untill you take a programming class!");
+                
+            }
+            break;
+        case 25:
+            System.out.println("Please enter your birth year.");
+            yourBirthYear=keyboard.nextInt();
+            
+            if (yourBirthYear > 1980)
+            {
+                System.out.println("You were born this way, which is lucky!");
+            }
+            else if (yourBirthYear <1900)
+            {
+                System.out.println("Wow you are really old, but stil lucky!");
+            }
+            else if (yourBirthYear > 2015)
+            {
+                System.out.println("Sorry you are not lucky because yoou weren't born yet.");
+            }
+        break;
+        case 100:
+            
+            char letterOne='p';
+            char letterTwo='e';
+            char letterThree='r';
+            char letterFour='f';
+            char letterFive='e';
+            char letterSix='c';
+            char letterSeven ='t';
+            int asciiSum;
+       
+            asciiSum= (int)letterOne + (int)letterTwo + (int)letterThree + (int)letterFour + (int)letterFive + (int)letterSix + (int)letterSeven; 
+       
+       
+            System.out.println("The word is 'perfect'.");
+            System.out.println("The sum of the word perfect is " + asciiSum + " .");
+        break;
+        
+        case 1:
+            System.out.println("This program has eneded have a great day!");
```

Comment:
Just a couple of observations. Some of the lines are really long. Splitting them up by continuing them or splitting into more System.out.println statements will make your code easier to read and easier to add/delete/edit. Also menus are usually lists. Adding some comments would also be good. You never know when you may want to reference this program again and comments will help you remember what you did.